src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppsInfrastructure.cs

```diff
@@ -297,6 +297,11 @@ public async Task ProcessResourceAsync(DistributedApplicationExecutionContext ex
 
                 await ProcessEnvironmentAsync(executionContext, cancellationToken).ConfigureAwait(false);
                 await ProcessArgumentsAsync(executionContext, cancellationToken).ConfigureAwait(false);
+
+                // Write a fake parameter for the container app environment
+                // so azd knows the Dashboard URL - see https://github.com/dotnet/aspire/issues/8449.
+                // This is temporary until a real fix can be made in azd.
+                AllocateParameter(_containerAppEnvironmentContext.Environment.ContainerAppDomain);
```

Comment:
Move this to the top of BuildContainerApp

Reply:
Done.